src/concepts/external_data_integration.md

```diff
@@ -0,0 +1,75 @@
+
+To prevent Fivetran from having access to our data warehouse, separate data destinations have been provisioned for Fivetran to dump imported data into. The data can be transformed and moved to different data destinations using bigquery-etl.
+
+When working in the **Fivetran_Dev** environment, data is written to `dev-fivetran`. All datasets are by default accessible to data engineers. So when working with sensitive data, it might be necessary to manually update the access permissions to the dataset. Ensure that the Fivetran service account has the necessary permissions to write data to the dataset when making updates.
+
+Once the development has finished, either manually delete previously created datasets or set an expiration date so that the datasets will be removed automatically after some time.
+
+## Setting up connectors
+
+This is a step-by-step guide for setting up a new connector in Fivetran, scheduling it via Airflow and transforming imported data via bigquery-etl:
+
+1. Log in to [Fivetran](https://fivetran.com) and select the environment the new connector should be created in.
+1. Click on the "Add Connector" button and search for the service data that should be imported from. If no connector is available for the service please check out ["Developing custom connectors"](#developing-custom-connectors) for writing custom connectors.
+1. Specify the name of the destination dataset Fivetran will dump data into.
+1. Configure the connector by providing API credentials and other required settings.
+1. Finish the setup and wait for the connection tests to pass.
+1. Once the connector has been created a historical data import will be automatically triggered. This import can take a few minutes up to several days depending on how much historical data is available.
+1. Specify the data transformation in [bigquery-etl](https://github.com/mozilla/bigquery-etl) or [private-bigquery-etl](https://github.com/mozilla/private-bigquery-etl) when working with sensitive data. Please follow the [general guide of creating a derived dataset](https://mozilla.github.io/bigquery-etl/cookbooks/creating_a_derived_dataset/). When writing the query, data will be queried from the dataset that was specified when the connector was set up. The connector documentation specifies what tables will be available and what the schemas will look like.
+   - Make sure to query from the correct data destination depending on the environment for which the connector was set up for.
+1. [Scheduling](https://github.com/mozilla/bigquery-etl/blob/main/docs/reference/scheduling.md) the ETL and the Fivetran import is also done in bigquery-etl. In the `metadata.yaml` file of the transformation queries. In addition to specifying the DAG, also specify the Fivetran tasks that the ETL depends on, for example:
```

Comment:
I was going to suggest linking to https://mozilla.github.io/bigquery-etl/reference/scheduling/ instead, but it looks like the code blocks are busted in that view.  Is that a known issue?

Reply:
Not a known issue. We should fix that, I opened https://github.com/mozilla/bigquery-etl/issues/2768